.github/workflows/tapioca-update.yml

```
@@ -0,0 +1,35 @@
+name: Tapioca Update
+on:
+  pull_request:
+    paths:
+      - .github/workflows/tapioca-update.yml
+  schedule:
+    - cron: '0 0 */3 * *' # Every three days
+env:
+  HOMEBREW_GITHUB_ACTIONS: 1
+  HOMEBREW_SORBET: 1
+jobs:
+  update-tapioca:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set up Homebrew
+        id: set-up-homebrew
+        uses: Homebrew/actions/setup-homebrew@master
+
+      - name: Configure Git user
+        uses: Homebrew/actions/git-user-config@master
+        with:
+          username: BrewTestBot
+
+      - name: Update Tapioca Definitions
+        run: |
```

```suggestion
        env:
          GITHUB_TOKEN: ${{secrets.HOMEBREW_GITHUB_API_TOKEN}}
        run: |
```

`hub` needs a token to access GitHub API but I don't know if this secret is available in this repo too, @MikeMcQuaid

---

We can use `GITHUB_TOKEN`, same as the other scheduled workflow:

https://github.com/Homebrew/brew/blob/5de4378a974c9f512c89892938ce949db8c08388/.github/workflows/spdx.yml#L23-L25